samples/meetings-app-icon-badging/csharp/README.md

```diff
@@ -0,0 +1,116 @@
+---
+page_type: sample
+description: This sample app shows how to send app icon badging in Teams meetings using targeted notifications API.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "12/06/2023 13:38:26 PM"
+urlFragment: officedev-microsoft-teams-samples-meetings-app-icon-badging-csharp
+---
+
+## App Icon Badging In Teams Meeting Notification
+
+This sample illustrates how to send app icon badging notification on tab icon in Teams meetings.
+
+## Included Features
+* Bots
+* In-Meeting Notifications
+* RSC Permissions
+* App Icon Badging
+
+## Interaction with app
+
+![App Icon Badging](AppIconBadgingInMeetings/Images/App_Icon_Badging.gif)
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 6.0
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+## Setup
+
+1. Register a new application in the [Azure Active Directory – App Registrations](https://go.microsoft.com/fwlink/?linkid=2083908) portal.
+
+2. Setup for Bot	
+	- Also, register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
+	- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+	- While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
+
+    > NOTE: When you create your app registration, you will create an App ID and App password - make sure you keep these for later.
+
+3. Setup NGROK
+      - Run ngrok - point to port 3978
+
+	```bash
+	# ngrok http 3978 --host-header="localhost:3978"
+	```
+4. Setup for code
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples\meetings-app-icon-badging\csharp` folder
```

Comment:
please check if it's correct?
samples\meetings-app-icon-badging\csharp

should it be like: samples/meetings-app-icon-badging/csharp

Reply:
fixed